src/style.css

```diff
@@ -1,3 +1,58 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --text-color: #00acdc;
+}
+
+.header {
+  font-family: Roboto, sans-serif;
+  width: 100%;
+  height: 60px;
```

Comment:
Please see the checklist this point:
the height should be set for nav links (not the header), take it from the design.
Remove height from here, it'll be calculated by height of header's children